README.md

```diff
@@ -1,2 +1,2 @@
 # ihs-website
-Website for Mr. Stutler's IHS courses
+Website for Mr. Stutler's IHS courses!
```

Comment:
You can ignore this... I just made a file change to trigger Github to build the site for my fork (to verify the fix)